### PR TITLE
master-qa-50753_1

### PR DESCRIPTION
**Pull request by chaowang-123**
https://issues.liferay.com/browse/LRQA-50753
Submit to master.

### CODE REVIEW

**Comment by chaowang-123** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 1]
```diff
@@ -567,13 +567,102 @@ definition {
...
+			GDPR.deleteStagedDatainApplicationCP(rowIndexNumber = "${rowIndexNum}");
+		}
+
+		BlogsNavigator.openBlogsAdmin(siteURLKey = "site-name-staging");
+
+		Blogs.viewNoEntryCP();
+
+		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "site-name-staging");
+
+		AssertTextEquals(
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase]: Please use the exist macro to replace.

**Review by Sharryxue — CHANGES_REQUESTED** [review]
Hello @chaowang-123 ,
Thanks for writing the test.
Only one change required. Please use the exist macro in test steps instead of using the function directly. Thanks

**Comment by chaowang-123** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

**Comment by Sharryxue** [comment]
No additional changes needed, just squashed the commits.
View [74b50148](https://github.com/Sharryxue/liferay-portal/pull/364/commits/74b50148fc9fd28d597e702838180ab192343047)

**Pull request closed without merging**